modules/logging-release-notes-6-0-0.adoc

```diff
@@ -0,0 +1,121 @@
+
+[id="logging-release-notes-6-0-0-known-issues"]
+== Known issues 
+
+None.
+
+[id="logging-release-notes-6-0-0-bug-fixes"]
+== Bug fixes
+
+=== Log Collection
+
+* Before this update, the `CollectorHighErrorRate` and `CollectorVeryHighErrorRate` alerts were still present. With this update, both alerts are removed in the Logging 6.0 release but might return in a future release. (link:https://issues.redhat.com/browse/LOG-3432[LOG-3432])
+
+* Before this update, the `must-gather` tool failed on clusters with non-AMD64 architectures such as Azure Resource Manager (ARM) or PowerPC. With this update, the tool now detects the cluster architecture at runtime and uses architecture-independent paths and dependencies. The detection allows `must-gather` to run smoothly on platforms like ARM and PowerPC. (link:https://issues.redhat.com/browse/LOG-5943[LOG-5943])
+
+* Before this update, the log level was set using a mix of structured and unstructured keywords that were unclear. With this update, the log level follows a clear, documented order, starting with structured keywords. (link:https://issues.redhat.com/browse/LOG-4805[LOG-4805])
+
+=== Log Storage
+
+* Before this update, the Loki Operator did not validate the Amazon Simple Storage Service (S3) endpoint URL format used in the storage secret. With this update, the S3 endpoint URL goes through a validation step that reflects on the status of the LokiStack. (link:https://issues.redhat.com/browse/LOG-5070[LOG-5070])
+
+* Before this update, the Loki Operator would crash if the `CredentialRequest` specification was registered in an environment without the `cloud-credentials-operator`. With this update, the `CredentialRequest` specification only registers in environments that are `cloud-credentials-operator` enabled. (link:https://issues.redhat.com/browse/LOG-5431[LOG-5431])
+
+* Before this update, the Loki Operator overwrote user annotations on the LokiStack Route resource, causing customizations to drop. With this update, the Loki Operator no longer overwrites Route annotations, fixing the issue. (link:https://issues.redhat.com/browse/LOG-5566[LOG-5566])
+
+=== Log Console
+
+* Before this update, an issue prevented selecting pods that no longer existed, even if they had generated logs. With this update, this issue has been fixed, allowing selection of such pods. (link:https://issues.redhat.com/browse/LOG-5287[LOG-5287])
+
+[id="logging-release-notes-6-0-0-CVEs"]
+== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2024-34397[CVE-2024-34397]
+
+[id="logging-release-notes-6-0-0-technology-preview-features"]
+== Technology Preview features
+
+* With this release, a Technology Preview of log forwarding through the OpenTelemetry Observability Framework is enabled. A new output type,` OTLP`, allows sending JSON-encoded log records using the OpenTelemetry data model and resource semantic conventions. (link:https://issues.redhat.com/browse/LOG-4225[LOG-4225])
```

Comment:
```suggestion
* With this release, introduces log forwarding using the OpenTelemetry. A new output type,` OTLP`, allows sending JSON-encoded log records using the OpenTelemetry data model and resource semantic conventions. (link:https://issues.redhat.com/browse/LOG-4225[LOG-4225])
```

Reply:
@JoaoBraveCoding If it still come under tech preview, I suppose it should come in the sentence as well. I'll make the rest of the changes.